src/features/signup/components/faqList.tsx

```diff
@@ -0,0 +1,35 @@
+import { FAQItem } from "./faqItem";
+
+export const FAQList = (): JSX.Element => (
+  <div className="mt-28 flex flex-col items-center justify-center">
+    <h1 className="mb-8">FAQ</h1>
+
+    <FAQItem
+      description="This found is focused on supporting Mexican projects working on reducing the gender gap, collaborating with local communities, and ReFi."
+      title="what is the focus of this round?"
+    />
+
+    <FAQItem
+      description="To participate in this round as a project or as a voter, you must be a Mexican community member and have on-chain proofs, such as: POAPs, Unlock Protocol, NFTs or EAS."
```

Comment:
you must be whitelisted by the organizers and old either a specific NFT, POAP or an EAS attestation